tests/test_security_fixes.py

```
@@ -0,0 +1,508 @@
+            assert 'functions_per_file' in complexity_metrics
+            assert 'lines_per_file' in complexity_metrics
+            assert complexity_metrics['total_files'] == 2
+            assert complexity_metrics['functions_per_file'] == 5.0
+            
+        except ImportError:
+            pytest.skip("ProjectMetrics not available")
+
+
+def run_security_tests():
+    """Run all security tests."""
+    import subprocess
```

Ruff (0.11.9)

Redefinition of unused `subprocess` from line 7

Remove definition: `subprocess`

(F811)